extension/src/paymentHandler/payment-handler.js

```diff
@@ -78,24 +71,19 @@ function _getPaymentHandlers(paymentObject) {
     paymentObject.custom.fields.createSessionRequest &&
     !paymentObject.custom.fields.createSessionResponse
   ) {
-    handlers.push(sessionRequestHandler)
+    const createSessionRequestObj = JSON.parse(
+      paymentObject.custom.fields.createSessionRequest
+    )
+    if (_requiresLineItems(createSessionRequestObj))
+      handlers.push(lineItemSessionRequestHandler)
+    else handlers.push(sessionRequestHandler)
   }
   if (
     paymentObject.custom.fields.getCarbonOffsetCostsRequest &&
     !paymentObject.custom.fields.getCarbonOffsetCostsResponse
   )
     handlers.push(getCarbonOffsetCostsHandler)
-  if (
-    paymentObject.custom.fields.makePaymentRequest &&
-    !paymentObject.custom.fields.makePaymentResponse
-  ) {
-    const makePaymentRequestObj = JSON.parse(
-      paymentObject.custom.fields.makePaymentRequest
-    )
-    if (_requiresLineItems(makePaymentRequestObj))
-      handlers.push(makeLineitemsPaymentHandler)
-    else handlers.push(execute)
-  }
+
```

Comment:
There won't be anymore makePaymentResponse custom field.
Can you please update this condition here in the line 88.
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/files#diff-33ad50b3bb6880513d8c4c007553160c74bf2be2f0356402d7c9bb540fd0e462R88

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1056/commits/254ca0ff56247f0a0be34c30162e4bb82200db3c